specification/cognitiveservices/Language.AnalyzeText/common.tsp

```diff
@@ -0,0 +1,80 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
```

Comment:
Since this shared by two sub services, it can be moved to `Common` under `cognitiveservices`.

Reply:
I tried that before but it didn't work with versioning.

Reply:
It doesnt seem like we need this file or have a need to make templates, since these operations can use operation templates in a pretty straightforward manner. However, if we ever want to have templates that are shared among the Conversations and TextAnalyze specifications, I think the following directory structure is worth a look at:
```
- Language/
  - AnalyzeText/
    - <files related to the analyze text spec>
  - Common/
    - <files containing common models or template defintions>
  - Conversations/
    - <files related to the conversations spec>
```